### DISCUSSION TITLE
Inconsistencias en el certificado

### DISCUSSION THREAD

**Discussion by JGranda11** [rramosp/ai4eng.v1#317] [00 - General]
Saludos!
Lo que pasa es es que quiero generar el certificado que se mencionaba al inicio del curso, pero cuando le doy en generar el certificado en la plataforma RLXMOOC, el PDF que se descarga no contiene toda la información:
![image](https://github.com/rramosp/ai4eng.v1/assets/112766584/54af06d9-9e15-4442-b651-37055fd99706)
Me gustaria saber como arreglarlo

**Comment by Felipe-RA** [comment]
Hola @JGranda11  

vamos a echarle un vistazo  a ver que está pasando

Esta semana no creemos poder arreglarlo, pero en cuanto podamos lo miramos

Gracias por el heads up
